crates/rolldown/tests/esbuild/dce/dce_of_using_declarations/artifacts.snap

```diff
@@ -19,8 +19,6 @@ await using await_asyncDispose_keep = { [Symbol.asyncDispose]() {
 } };
 using undef_keep = undefined;
 await using await_undef_keep = undefined;
-const Symbol_dispose_remove = Symbol.dispose;
-const Symbol_asyncDispose_remove = Symbol.asyncDispose;
 console.log(null_keep, undef_keep);
 
```

Comment:
```
// entry.js
using null_keep = null;
await using await_null_keep = null;
using throw_keep = {};
using dispose_keep = { [Symbol.dispose]() {
  console.log("side effect");
} };
await using await_asyncDispose_keep = { [Symbol.asyncDispose]() {
  console.log("side effect");
} };
using undef_keep = void 0;
await using await_undef_keep = void 0;
console.log(
  null_keep,
  undef_keep
);
```
Still not same as esbuild, but not relevant to this pr, these two stmt is removed either in esbuild snapshot